src/components/uploader/js/Uploader.js

```diff
@@ -200,20 +191,11 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     };
 
     fluid.uploader.updateTotalProgress = function (that) {
-        // Preservered for backwards compatibility, should be refactored post v1.5
         var batch = that.queue.currentBatch;
         var totalPercent = fluid.uploader.derivePercent(batch.totalBytesUploaded, batch.totalBytes);
-        var numFilesInBatch = batch.files.length;
-        var fileLabelStr = fluid.uploader.fileOrFiles(that, numFilesInBatch);
-
         var uploadingSize = batch.totalBytesUploaded + that.queue.sizeOfUploadedFiles();
 
         var totalProgressStr = fluid.stringTemplate(that.options.strings.progress.totalProgressLabel, {
-            curFileN: batch.fileIdx,
-            totalFilesN: numFilesInBatch,
-            fileLabel: fileLabelStr,
-            currBytes: fluid.uploader.formatFileSize(batch.totalBytesUploaded),
```

Comment:
@amb26 as of Infusion 1.5 these values were no longer used in any of the default strings, but they were left in place for backwards compatibility incase a user had custom strings here. 

See the strings section from:
http://docs.fluidproject.org/infusion/development/UploaderAPI.html#uploader-options